### PR TITLE
Update urls

### PR DESCRIPTION
**Pull request by willkg**
* update to new readthedocs urls
* switch to https

r?

### CODE REVIEW

File: docs/conventions.rst [hunk 1]
```diff
@@ -11,7 +11,7 @@ Coding conventions
 
 We follow most of the practices as detailed in the `Mozilla webdev
 bootcamp guide
-<http://mozweb.readthedocs.org/en/latest/coding.html>`_.
+<https://mozweb.readthedocs.io/en/latest/coding.html>`_.
```
Comment (jwhitlock) [inline docs/conventions.rst]: 404. I think the URL should be https://mozweb.readthedocs.io/en/latest/guide/development_process.html

File: docs/searchchapter.rst [hunk 2]
```diff
@@ -110,7 +110,7 @@ These settings explained:
 There are a few other settings you can set in your
 ``kitsune/settings_local.py`` file that override ElasticUtils defaults.  See
 `the ElasticUtils docs
-<http://elasticutils.readthedocs.org/en/latest/installation.html#configure>`_
+<https://elasticutils.readthedocs.io/en/latest/installation.html#configure>`_
```
Comment (jwhitlock) [inline docs/searchchapter.rst]: Link is OK, but I think https://elasticutils.readthedocs.io/en/latest/django.html#configuration is a better target.

File: docs/conventions.rst [hunk 3]
```diff
@@ -63,7 +63,7 @@ Git resources and tools
 -----------------------
 
 See `Webdev bootcamp guide
-<http://mozweb.readthedocs.org/en/latest/git.html#git-and-github>`_
+<https://mozweb.readthedocs.io/en/latest/git.html#git-and-github>`_
```
Comment (jwhitlock) [inline docs/conventions.rst]: 404. https://mozweb.readthedocs.io/en/latest/reference/git_github.html looks correct.

**Comment by willkg** [comment]
I appreciate you finding correct urls. I lazily just did a couple of search-and-replaces.

I wonder if the incredible wrongness of some of these urls suggests the documentation in those areas is also horribly wrong.

**Comment by willkg** [comment]
^^^ That updates the urls you pointed out as being wrong.

**Comment by jwhitlock** [comment]
+r. I don't have commit access, and don't want it, so someone else needs to merge.

**Comment by safwanrahman** [comment]
@r+ from my end. @willkg You may have commit access or @glogiotatidis can merge

**Comment by glogiotatidis** [comment]
Thanks folks!

**Pull request merged**